### PR TITLE
Scaffold

### PR DESCRIPTION
**Pull request by awendt**
This follows [Creating a Javascript action](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action).

Things to note:

> Checking in your `node_modules` directory can cause problems. As an alternative, you can use a tool called `@vercel/ncc` to compile your code and modules into one file used for distribution.

This is what we do here, build script is `npm run build`.

## Next steps

- add CI to [check if `dist/index.js` is up-to-date](https://github.com/actions/github-script/blob/main/.github/workflows/check-dist.yml)
- add logic and tests
- Dependabot

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Remove stale projects
+description: |
+  Determine stale projects and remove them
+
+inputs:
+  current-projects:
+    description: "List of projects the given pull request is assigned to"
```
Comment (jtsaito) [inline action.yml:7]:
```suggestion
    description: List of projects the given pull request is assigned to
```

File: action.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    required: true
+  github-token:
+    description: The GitHub token used to create an authenticated client
+    default: ${{ github.token }}
+    required: false
+  target-projects:
+    description: "List of projects the given pull request should be assigned to"
```
Comment (jtsaito) [inline action.yml:14]:
```suggestion
    description: List of projects the given pull request should be assigned to
```

File: index.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+
+async function run() {
+  try {
+    // `who-to-greet` input defined in action metadata file
+    const nameToGreet = core.getInput('who-to-greet');
```
Comment (jtsaito) [inline index.js:7]: I think this inputs must match the definition in the YAML.

File: action.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    required: true
+
+runs:
+  using: 'node12'
```
Comment (fmasuhr) [inline action.yml:18]:
```suggestion
  using: node12
```

File: action.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  main: 'dist/index.js'
```
Comment (fmasuhr) [inline action.yml:19]:
```suggestion
  main: dist/index.js
```

**Review by fmasuhr — APPROVED** [review]

File: package.json [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+  "version": "0.0.1",
+  "description": "Determine stale projects and remove them",
+  "main": "index.js",
+  "scripts": {
+    "build": "ncc build index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "dependencies": {
+    "@actions/core": "^1.6.0",
+    "@actions/github": "^5.0.0"
```
Comment (fmasuhr) [inline package.json:12]: I think it is not necessary to pin them here as we do use the lockfile. 
```suggestion
    "@actions/core": "*",
    "@actions/github": "*"
```

File: package.json [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+  },
+  "devDependencies": {
+    "@vercel/ncc": "^0.32.0"
```
Comment (fmasuhr) [inline package.json:15]:
```suggestion
    "@vercel/ncc": "*"
```

File: index.js (hunk 3 above)
Comment (awendt) [reply index.js:7]: I know, but this is not expected to succeed just yet. I wanted to get the boilerplate done before sending a pull request with the real code.

**Review by jtsaito — APPROVED** [review]

**Pull request merged**